<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataWidget</class>
 <widget class="QWidget" name="DataWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_name">
         <property name="text">
          <string>Nom:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_nbPlayer">
         <property name="text">
          <string>Nombre de joueurs:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_nbMJ">
         <property name="text">
          <string>Nombre de MJ:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_nbInvestigations">
         <property name="text">
          <string>Nombre d'enquêtes:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_author">
         <property name="text">
          <string>Auteur: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_date">
         <property name="text">
          <string>Date de Création:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_length">
         <property name="text">
          <string>Durée:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_nbEvents">
         <property name="text">
          <string>Nombre d'événements:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="pb_showInvest">
       <property name="text">
        <string>Voir les enquêtes</string>
       </property>
       <property name="icon">
        <iconset resource="../ressources.qrc">
         <normaloff>:/icones/glass</normaloff>:/icones/glass</iconset>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="pb_showDiary">
       <property name="text">
        <string>Voir les événements</string>
       </property>
       <property name="icon">
        <iconset resource="../ressources.qrc">
         <normaloff>:/icones/log</normaloff>:/icones/log</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../ressources.qrc"/>
 </resources>
 <connections/>
</ui>
